test/extended/images/prune.go

```diff
@@ -41,7 +41,7 @@ var _ = g.Describe("[images] prune images", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		g.By(fmt.Sprintf("give a user %s a right to prune images with %s role", oc.Username(), "system:image-pruner"))
-		err = oc.AsAdmin().Run("adm").Args("policy", "add-cluster-role-to-user", "system:image-pruner", oc.Username()).Execute()
+		err = oc.AsAdmin().NoNamespace().Run("adm").Args("policy", "add-cluster-role-to-user", "system:image-pruner", oc.Username()).Execute()
```

Comment:
got it, and we don't want to break that. I will name it `WithNoNamespace()` then but that is a minor nit